lib/rgeo/impl_helper/basic_line_string_methods.rb

```diff
@@ -64,9 +62,7 @@ def end_point
       end
 
       def closed?
-        unless defined?(@closed)
-          @closed = @points.size > 2 && @points.first == @points.last
-        end
+        @closed = @points.size > 2 && @points.first == @points.last unless defined?(@closed)
```

Comment:
```suggestion
        return @closed if defined?(@closed)

        @closed = @points.size > 2 && @points.first == @points.last unless defined?(@closed)
```

(note that the suggestion is not enough, I couldn't select the whole method)